saleor/graphql/schema.graphql

```diff
@@ -2930,7 +2937,7 @@ type Product implements Node {
   thumbnail(size: Int): Image
   availability: ProductPricingInfo @deprecated(reason: "DEPRECATED: Will be removed in Saleor 2.10, Has been renamed to `pricing`.")
   pricing: ProductPricingInfo
-  isAvailable: Boolean
+  isAvailableInCountry: Boolean
```

Comment:
`isAvailable(country: CountryCode): Boolean` - for easier adoption country could be nullable if we have some way to return a default if none is provided.